tasks/build/verify_translations.js

```diff
@@ -45,12 +44,30 @@ export default function (grunt) {
   });
 }
 
-function verifyTranslations(uiI18nObj, parsePaths)
+function verifyTranslations(uiI18nObj)
 {
-  return uiI18nObj.getAllTranslations()
-  .then(function (translations) {
-    return i18nVerify.getTranslationKeys(parsePaths)
-    .then(function (translationKeys) {
+  const translationPatterns = [
+    { regEx: 'translate=\"([^\"]+)\"',
+      parsePaths: [fromRoot('/src/core_plugins/kibana/*.html')] },
```

Comment:
Did you try the grunt plugin? The one issue I have with this is that it won't catch usages of the translate filter or the translate service in JS.